airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -177,13 +189,15 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
                           <FormattedMessage id="admin.dockerImageTag" />
                         </Label>
                       }
+                      error={!!meta.error && meta.touched}
+                      message={meta.touched && meta.error && formatMessage({ id: meta.error })}
```

Comment:
nit: any reason to use `formatMessage()` here and `<FormattedMessage>` on lines 147 and 172?

Reply:
Nope, I missed this when I was cleaning the others from a copy/paste!

Reply:
Nope!  I missed some cleanup from a copy/paste.  Thanks.